Modules/getpath.c

```diff
@@ -605,6 +640,15 @@ calculate_program_full_path(const _PyCoreConfig *core_config,
     if (program_full_path[0] != SEP && program_full_path[0] != '\0') {
         absolutize(program_full_path);
     }
+#if defined(__CYGWIN__) || defined(__MINGW32__)
+    /* For these platforms it is necessary to ensure that the .exe suffix
+     * is appended to the filename, otherwise there is potential for
+     * sys.executable to return the name of a directory under the same
+     * path (bpo-28441).
+     */
+    if (program_full_path[0] != '\0')
+        add_exe_suffix(program_full_path);
```

Comment:
Use brackets `{}` for new code.

Reply:
Ah, I missed that updated to PEP-7.  Just as well.  I don't like if statements without braces either.